prow/cmd/mkpj/main.go

```diff
@@ -80,6 +172,18 @@ func main() {
 		logrus.WithError(err).Fatal("Error loading config.")
 	}
 
+	var secretAgent *secret.Agent
+	if o.github.TokenPath != "" {
+		secretAgent = &secret.Agent{}
+		if err := secretAgent.Start([]string{o.github.TokenPath}); err != nil {
+			logrus.Fatalf("Failed to start secret agent: %v", err)
+		}
+	}
+	o.githubClient, err = o.github.GitHubClient(secretAgent, false)
```

Comment:
Why not expose the dry-run flag?

Reply:
Because it doesn't make any difference, as this CLI never does any mutating changes to Github